### PR TITLE
[1.1.1.1] Update DNS over Discord docs to include changing DNS provider

### PR DESCRIPTION
**Pull request by MattIPv4**
cc https://github.com/MattIPv4/DNS-over-Discord/pull/57

### CODE REVIEW

**Comment by fariborz4** [comment]
> cc https://github.com/MattIPv4/DNS-over-Discord/pull/57

Ok

**Comment by MattIPv4** [comment]
👋 Hey @marciocloudflare, would it be possible to get this PR looked at? Cheers!

**Comment by marciocloudflare** [comment]
Hey @MattIPv4. Our dev docs are focused on Cloudflare products only. Please resubmit a PR with the names of any non-Cloudflare products and respective images / animated GIFs removed. Thanks!

**Comment by MattIPv4** [comment]
The bot supports alternate DNS providers though to compare DoH responses. Not sure how I can remove that when it's documenting functionality?

**Comment by MattIPv4** [comment]
I'm happy to update the GIFs to demonstrate switching between different 1dot options, but the other DNS providers are in the dropdown and so I feel should be listed in the documentation

**Comment by marciocloudflare** [comment]
Could you remove all the images then? We've been removing images from the dev docs and only adding whenever there's a situation that really calls for it - like something in the UI that might be confusing. I don't think that's the case here

**Comment by MattIPv4** [comment]
Ack sounds good and makes sense. Will get them removed shortly 👍

**Comment by MattIPv4** [comment]
Images removed, apologies for the delay ✅

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 1]
```diff
@@ -76,11 +76,24 @@ You can refresh the DNS lookup results by clicking the Refresh button. Clicking
...
+
+<details>
+<summary>Supported DNS providers</summary>
+<div>
+
+* [1.1.1.1 (Cloudflare)](https://developers.cloudflare.com/1.1.1.1/)
+* [1.1.1.2 (Cloudflare Malware Blocking)](https://developers.cloudflare.com/1.1.1.1/setup/#1111-for-families)
+* [1.1.1.3 (Cloudflare Malware + Adult Content Blocking)](https://developers.cloudflare.com/1.1.1.1/setup/#1111-for-families)
+* [8.8.8.8 (Google)](https://developers.google.com/speed/public-dns)
+* [9.9.9.9 (Quad9)](https://www.quad9.net/)
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
By default, the DNS over Discord bot uses Cloudflare's 1.1.1.1 DNS service. You can run the DNS lookup with alternate DNS providers by selecting the dropdown below the result. This shows you a list of available providers. Selecting a new provider updates the results in the message. Any user can change the DNS provider.
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 2]
```diff
@@ -167,11 +180,24 @@ The `/multi-dig` command also provides a refresh button below each set of DNS re
 
 As with the `/dig` command, any user can press the refresh button to refresh the displayed DNS results, including for DNS queries that had previously failed.
 
-<div class="medium-img">
+### Changing DNS provider
+
+Much like the `/dig` command, the `/multi-dig` command also allows for the DNS provider being used for the lookup to be changed. The menu will appear after each set of DNS results (or after each block of results if more than 10 record types are requested).
+
+This menu can be used be any user to change the DNS provider used for the lookup, like with the `/dig` command.
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
Like the `/dig` command, you can change the DNS provider when using the `/multi-dig` command. The menu appears after each set of DNS results (or after each block of results if more than 10 record types are requested).

This menu can be used be any user to change the DNS provider used for the lookup.
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 3]
```diff
@@ -167,11 +180,24 @@ The `/multi-dig` command also provides a refresh button below each set of DNS re
...
+<div>
+
+* [1.1.1.1 (Cloudflare)](https://developers.cloudflare.com/1.1.1.1/)
+* [1.1.1.2 (Cloudflare Malware Blocking)](https://developers.cloudflare.com/1.1.1.1/setup/#1111-for-families)
+* [1.1.1.3 (Cloudflare Malware + Adult Content Blocking)](https://developers.cloudflare.com/1.1.1.1/setup/#1111-for-families)
+* [8.8.8.8 (Google)](https://developers.google.com/speed/public-dns)
+* [9.9.9.9 (Quad9)](https://www.quad9.net/)
 
 </div>
+</details>
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 4]
```diff
@@ -202,14 +228,24 @@ Example:
 
 ### `privacy` command
 
-The `/privacy` command displays the privacy policy notice for using the 1.1.1.1 DNS over Discord bot. This notice can also be viewed at [https://dns-over-discord.v4.wtf/privacy](https://dns-over-discord.v4.wtf/privacy).
+The `/privacy` command displays the Privacy Policy notice for using the 1.1.1.1 DNS over Discord bot. This notice can also be viewed at [https://dns-over-discord.v4.wtf/privacy](https://dns-over-discord.v4.wtf/privacy).
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
The `/privacy` command displays the Privacy Policy notice for using the 1.1.1.1 DNS over Discord bot. You can also [refer to the Privacy Policy page](https://dns-over-discord.v4.wtf/privacy) to access it.
```

File: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md [hunk 5]
```diff
@@ -202,14 +228,24 @@ Example:
...
 
 Example:
 
 ```txt
 /privacy
 ```
 
+### `terms` command
+
+The `/terms` command displays the Terms of Service notice for using the 1.1.1.1 DNS over Discord bot. This notice can also be viewed at [https://dns-over-discord.v4.wtf/terms](https://dns-over-discord.v4.wtf/terms).
```
Comment (marciocloudflare) [inline content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md]:
```suggestion
The `/terms` command displays the Terms of Service notice for using the 1.1.1.1 DNS over Discord bot. You can also [refer to the Terms of Service page](https://dns-over-discord.v4.wtf/terms) to access it.
```

**Review by marciocloudflare — APPROVED** [review]

**Comment by marciocloudflare** [comment]
Good morning @MattIPv4. I've made a few edits to make this more inline with our style guide and merged it. Thanks! :)

**Comment by MattIPv4** [comment]
Thank you very much! 🧡

**Pull request merged**